types/enzyme-async-helpers/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for enzyme-async-helpers 0.9
+// Project: https://github.com/zth/enzyme-async-helpers
+// Definitions by: Kim Ehrenpohl <https://github.com/kimehrenpohl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { ReactWrapper, EnzymeSelector } from 'enzyme';
+
+export function waitForState(
+    wrapper: ReactWrapper,
+    stateValidationFn: (state: object) => boolean,
+    config?: Config
+): Promise<void>;
+
+export function waitForElement(
+    wrapper: ReactWrapper,
+    elementSelector: EnzymeSelector,
+    config?: Config
+): Promise<void>;
+
+export function waitForProps(
+    wrapper: ReactWrapper,
+    propsValidationFn: (props: object) => boolean,
+    config?: Config
+): Promise<void>;
+
+export interface Config {
+    interval: number; // Default: 50, how often to check for validity
```

Comment:
It looks like all these properties can be optional since [the method has `||` coalescing](https://github.com/zth/enzyme-async-helpers/blob/master/src/wait.js#L48).

Reply:
Thanks for pointing this out. This is should be fixed in c68f3725aa100d130da3d5fa605cf852c09a1fe3 .